Gems/Atom/RHI/Code/Source/RHI/PipelineStateCache.cpp

```diff
@@ -316,7 +321,11 @@ namespace AZ
                     if (!threadLibraryEntry.m_library)
                     {
                         Ptr<PipelineLibrary> pipelineLibrary = Factory::Get().CreatePipelineLibrary();
-                        RHI::ResultCode resultCode = pipelineLibrary->Init(*m_device, globalLibraryEntry.m_serializedData.get());
+                        RHI::PipelineLibraryDescriptor pipelineLibraryDescriptor;
+                        pipelineLibraryDescriptor.m_serializedData = globalLibraryEntry.m_serializedData.get();
+                        pipelineLibraryDescriptor.m_filePath = m_libraryFileNames[handle.GetIndex()];
+                        
+                        RHI::ResultCode resultCode = pipelineLibrary->Init(*m_device, pipelineLibraryDescriptor);
```

Comment:
2nd place that duplicates the copying from 2 vectors to 1 PipelineLibraryDescriptor.